src/sentry/static/sentry/app/components/charts/components/xAxis.jsx

```diff
@@ -2,16 +2,22 @@ import moment from 'moment';
 
 import theme from 'app/utils/theme';
 
+const TRUNCATION_LENGTH = 80;
+
 export default function XAxis({isGroupedByDate, interval, ...props} = {}) {
   const axisLabelFormatter = isGroupedByDate
     ? (value, index) => {
-        const format = interval === 'hour' ? 'LT' : 'MMM Do';
-        return moment
-          .utc(value)
-          .local()
-          .format(format);
+      const format = interval === 'hour' ? 'LT' : 'MMM Do';
```

Comment:
this first ternary is a prettier change, but I have the same version as in package.json so I think it should be fine. I can revert the spacing if we want.